ros2_control_test_assets/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2_control_test_assets</name>
+  <version>0.0.0</version>
```

Comment:
```suggestion
  <version>0.1.3</version>
```
I'd need to fix this up before releasing, easier to have it synch-ed up from the beginning